universal-application-tool-0.0.1/test/repository/WithPostgresContainer.java

```diff
@@ -33,7 +33,7 @@ protected Application provideApplication() {
              * jdbc: the standard java database connection uri scheme
              * tc: Testcontainers - the tool that starts a new container per test.
              * postgresql: which container to start
-             * 9.6.8: which version of postgres to start
+             * 12.5: which version of postgres to start
```

Comment:
Thanks! I saw this when I first deleted `WithPostgresContainer` and totally forgot when I reverted it.